web_remember_tree_column_width/static/src/scss/main.scss

```diff
@@ -0,0 +1,8 @@
+th.o_column_sortable {
```

Comment:
@kanda999 Just FYI, there has been an update on the 16.0 module to change this key.